wannierberri/__evaluate.py

```diff
@@ -212,7 +213,12 @@ def paralfunc(Kpoint,_system,_grid,npar_k,fftlib):
         time1 = time()
         print("time1 = ",time1-time0)
         if not (restart and i_iter==0):
-            result_all.write(fout_name+"-{}"+suffix+"_iter-{0:04d}.dat".format(i_iter+start_iter))
+            _name = fout_name+"-{}"+suffix+"_iter-{0:04d}".format(i_iter+start_iter)
+            if write_txt:
+                result_all.write(_name+".dat")
+            if write_bin:
+                result_all.save(_name)
```

Comment:
This  non-standardized behavour has some sense. Namely a text file can have any extension, therefore it is not reasonable to encode `'.dat'` into  `.write()` , because one may which to write it to a file with different extension (or no extension at all).

In turn`'.npz'` points to a specific file format. Therefore, if we in future wish to  change the formast (e.g. to `'.npy'` of `'.pickle'`, then change should be contained within `.save()`

Reply:
I see, thanks for the explanation.